.github/workflows/release.yml

```diff
@@ -48,8 +114,15 @@ jobs:
           git config --global user.email 'leanspace-bot@users.noreply.github.com'
           git status
           git add -A
-          git diff-index --quiet HEAD || git commit -m "Update docs"
-          git push
+          git commit -m "Update docs"
+          if git diff-index --quiet HEAD
+            then
+              git tag ${{ github.event.inputs.release-version }} HEAD
+              git push --follow-tags
```

Comment:
this needs to also be present in the else, otherwise the tag is not present if there's no change in the documentation, right?

Reply:
Yes. I'll make the changes. Thanks :slightly_smiling_face: